iOS/classes/BleModule.swift

```diff
@@ -376,17 +376,34 @@ public class BleClientManager : NSObject {
         }
 
         var timeout: Int? = nil
-
+        
+        var connectOptions: [String: AnyObject] = [:]
+        
+        
         if let options = options {
             timeout = options["timeout"] as? Int
+            
+            if let isConnection = options["notifyOnConnection"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnConnectionKey] = isConnection as AnyObject
+            }
+            if let isDisconnection = options["notifyOnDisconnection"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnDisconnectionKey] = isDisconnection as AnyObject
+            }
+            if let isNotification = options["notifyOnNotification"] as? Bool {
+                connectOptions[CBConnectPeripheralOptionNotifyOnNotificationKey] = isNotification as AnyObject
+            }
```

Comment:
Could you please add remaining parameters with proper iOS version checks from https://developer.apple.com/documentation/corebluetooth/cbcentralmanager/peripheral_connection_options?